.github/workflows/refresh-lockfiles.yml

```diff
@@ -100,6 +100,7 @@ jobs:
           labels: |
             New: Pull Request
             Bot
+            benchmark_this
```

Comment:
The benchmarking is only being applied to the iris repo, but this workflow is intended to be, and is being, used in multiple other repos

Reply:
I had assumed this wasn't a problem, given `New: Pull Request` is also not a label that we use on all repos either.